### ISSUE TITLE
Zend\Mail: email parsing doesn't support base64 encoded comma inside name

### ISSUE THREAD

**Issue by GeeH** [zendframework/zend-mail#101]
This issue has been moved from the `zendframework` repository as part of the bug migration program as outlined here - http://framework.zend.com/blog/2016-04-11-issue-closures.html

___
Original Issue: https://api.github.com/repos/zendframework/zendframework/issues/7363
User: @echampet
Created On: 2015-03-24T15:20:50Z
Updated At: 2015-11-06T21:02:46Z
Body
Hi

#6267 isn't enough. 
in https://github.com/zendframework/zf2/blob/master/library/Zend/Mail/Header/AbstractAddressList.php#L44
str_getcsv should happen before iconv_mime_decode

An exemple of "broken" header 
```
From: =?utf-8?B?QUFBLCB0ZXN0IHdpdGggw6nDoCTDuQ==?= <aaaa@bbbb.com>
```

___
### Comment
User: @samsonasik
Created On: 2015-03-25T02:16:43Z
Updated At: 2015-03-25T02:16:43Z
Body
PR ?

___
### Comment
User: @echampet
Created On: 2015-03-25T13:42:01Z
Updated At: 2015-03-25T13:42:01Z
Body
@samsonasik, I'm working on many patches right now

___

**Comment by michalbundyra** [comment]
This repository has been closed and moved to laminas/laminas-mail; a new issue has been opened at https://github.com/laminas/laminas-mail/issues/55.
